pkg/typedef/columns.go

```diff
@@ -81,11 +81,16 @@ func (c Columns) Names() []string {
 	return names
 }
 
-func (c Columns) Remove(idx int) Columns {
-	out := c
-	copy(out[idx:], out[idx+1:])
-	out[len(out)-1] = nil
-	return out[:len(c)-1]
+func (c Columns) RemoveByColumnDef(column *ColumnDef) Columns {
```

Comment:
Since we don't have other `Remove` methods, could you please short it's name down to `Remove`